backend/src/Manager/StoreOwner/StoreOwnerProfileManager.php

```diff
@@ -102,4 +102,9 @@ public function getStoreProfileByStoreId($userId): ?array
     {
         return $this->storeOwnerProfileEntityRepository->getStoreProfileByStoreId($userId);
     }
+
+    public function getStoreOwnerProfile($id)
```

Comment:
function's return type + variable hint type must be mentioned.

Reply:
done